auth/only_api.go

```diff
@@ -109,7 +101,13 @@ func NewOnlyAPIUserManager(config *evergreen.OnlyAPIAuthConfig) (gimlet.UserMana
 	}
 	cache, err := usercache.NewExternal(opts)
 	if err != nil {
```

Comment:
You can just call `catcher.Wrap` without checking for `err != nil` (it'll no-op if the error is nil).